### PR TITLE
Enhance README.md for adding '.json' to the code block.

### PR DESCRIPTION
**Pull request by lamtrinhdev**
Enhance README.md for adding '.json' to the code block.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -580,7 +580,7 @@ defineInlineTest(transform, /* ... */)
 
 It's recommended that you set up your codemod project to all debugging via the VSCode IDE. When you open your project in VSCode, add the following configuration to your launch.json debugging configuration.
 
-```
+```json
```
Comment (Daniel15) [inline README.md:583]: Can you please change it to `jsonc` (JSON with comments) instead? That should make it highlight the comments properly.

Comment (lamtrinhdev) [reply README.md]: Dear @Daniel15 ,

Thank you for your support. I updated. Please help me to review it.

Thanks,
Lam

**Comment by lamtrinhdev** [comment]
Dear @Daniel15 ,

Thank you for your support. I updated follow the comment. Please help me to review it.

Thanks,
Lam

**Comment by Daniel15** [comment]
Sorry for the delay in replying. Thanks for your contribution!

**Pull request merged**